src/DayPickerInput.js

```diff
@@ -78,21 +78,28 @@ export default class DayPickerInput extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const { month, value } = this.state;
-    const hasDifferentValue = nextProps.value !== value;
+    const { dayPickerProps: { month }, value } = this.props;
```

Comment:
To make the code easier to understand, I usually avoid destructuring an object at the second level: could you please here just use:

```js
const { dayPickerProps, value } = this.props;
```

and update the code below? Thanks!

Reply:
Will do!